### ISSUE TITLE
Lab 6 lvl 3 Unable to Override put method properly

### ISSUE THREAD

**Issue by rshalinisai** [nus-cs2030/2223-s2#613]
I'm getting the error that I'm not Overriding the put method properly but I'm unsure what's wrong?

This is the error message I get and my put method in the Student class:

![photo1679794880](https://user-images.githubusercontent.com/122519417/227750742-1a6c2e48-4ca5-41c7-9f1d-e6ca86145b6b.jpeg)

My Student class extends KeyableMap:

![photo1679794930](https://user-images.githubusercontent.com/122519417/227750761-7a21ef98-bbdd-4f57-a075-3513b9954060.jpeg)

This is the put method in KeyableMap:

![image](https://user-images.githubusercontent.com/122519417/227750774-b4a55d06-fef1-4605-a5f2-ab684f04a617.png)

**Comment by Yihao2001** [comment]
can just try removing the @Override

**Comment by siruislybro** [comment]
In your Student and Module class, you need to declare the generic type when extending from KeyableMap

**Comment by ghost1nthemachine** [comment]
yep, as stated above for your class Student you need to declare the generic type, 

Class Student extends KeyableMap<Module> 

need to declare for your Module class too

**Comment by viethaaaa** [comment]
hi for your Student class you need to declare the generic type i.e. class Student extends KeyableMap<Module>
